.github/workflows/tests.yaml

```diff
@@ -0,0 +1,56 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  tests-on-phpunit-10:
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix:
+        os:
+          - "ubuntu-latest"
+        php:
+          - 8.2
+          - 8.3
```

Comment:
Looking at the jobs, it looks like this is only testing two different minor releases of Laravel 9, not 10 or 11 at all: https://github.com/carsdotcom/laravel-json-schema/actions/runs/9909395946

In the single table project, I ended up using the matrix to test every Laravel on modern PHP, then hand maintaining a list of lower Laravels that are even compatible with lower PHPs:
https://github.com/carsdotcom/single-table-inheritance/blob/main/.github/workflows/ci.yml and that seems to work a little better: https://github.com/carsdotcom/single-table-inheritance/actions/runs/9767346607

Then I don't control the Phpunit version, I let Laravel and testbench pick a compatible combo.